docs/en/observability/profiling-use-a-proxy.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[profiling-use-a-proxy]]
+= Use a proxy with the host-agent
+
+++++
+<titleabbrev>Use a proxy</titleabbrev>
+++++
+
+beta::[]
+
+In some cases, your infrastructure host-agent installation needs to use an HTTP proxy to reach Elastic Cloud.
+In these cases, you can use the `HTTPS_PROXY` environment variable to configure a proxy used by the host-agent.
+The connection to the backend ({ecloud}) will be tunneled through the proxy (no MITM-TLS) using the CONNECT method. 
+Basic authentication is supported. 
+This can be useful in environments with security policies or network restrictions that block direct connections to outside hosts. 
+
+IMPORTANT:  You need to set the `HTTPS_PROXY` environment variable in the machines that will run the host-agents. If you set the environment variable after the host-agent is installed, you may need to reinstall the host-agent.
+
+The following example shows how to use the `HTTPS_PROXY` environment variable:
+
+```
+export HTTPS_PROXY=http://username:password@proxy:port
+```
```

Comment:
This should be updated to an asciidoc code block: https://github.com/elastic/docs#code-blocks. Triple backticks do render in asciidoc, but they're pretty ugly with our stylesheet.

Reply:
Looks like there are a few others that should be updated as well. BTW, you don't need a million dashes like the docs show. Four is the minimum:

```
[source,bash]
----
export HTTPS_PROXY=http://username:password@proxy:port
----
```